src/services/scriptingservice.cpp

```diff
@@ -1986,6 +1995,37 @@ bool ScriptingService::writeToFile(const QString &filePath,
     return true;
 }
 
+/**
+ * Read text from a file
+ *
+ * @param filePath
+ * @return the file data or null if the file does not exist
+ */
+QString ScriptingService::readFromFile(const QString &filePath) const{
+    if (filePath.isEmpty()) return Q_NULLPTR;
```

Comment:
Please generally use braces for all if statements.

Reply:
I just changed it to fit the rest of the code style :smile: I'll keep that in mind.